plugin.rb

```diff
@@ -1048,6 +1055,10 @@ def schema
       end
     end
 
+    def groups
+      render json: Group.select(:id, :name)
```

Comment:
This bypasses security, probably not mega risky as it is admin only, but why not rely on existing group api?

Reply:
Agreed, no reason to not use existing group api.